bundle/backstage.io/manifests/backstage-operator.clusterserviceversion.yaml

```
@@ -35,7 +35,7 @@ metadata:
           }
         }
       ]
-    createdAt: "2025-03-19T17:41:51Z"
+    createdAt: "2025-03-31T10:06:52Z"
```

```suggestion
    createdAt: "2025-03-19T17:41:51Z"
```

Does not need to be changed.